ckan/templates/macros/form/resource_validation_errors.html

```diff
@@ -17,21 +17,25 @@
 
 
 {# (canada fork only): error -> danger BS version #}
-{% macro resource_validation_errors(errors={}, type="danger", classes=[], pkg_name='', helpers={}) %}
+{% macro resource_validation_errors(errors={}, type="danger", classes=[], pkg_name='', dataset_type='dataset', helpers={}) %}
```

Comment:
Hmm. If we don't have `h` in our macros maybe we're better off passing in the one helper we need? Or at least use the name `h` because that's how it looks in every other template.

Also the example and parameter descriptions in the comment above should be updated.